test/import-http.bats

```diff
@@ -0,0 +1,70 @@
+load helpers
+
+function teardown() {
+    cleanup
+    rm -rf img || true
+    rm -rf reference || true
+    rm -rf dest || true
+}
+
+function setup() {
+    mkdir reference
+    mkdir dest
+    rm -f favicon.ico
+    wget https://www.cisco.com/favicon.ico -O reference/favicon_orig.ico
+    echo "fake favicon" > reference/favicon_fake.ico
+    mkdir img
+    cat > img/stacker.yaml <<EOF
+img:
+    from:
+        type: docker
+        url: docker://centos:latest
+    import:
+        - https://www.cisco.com/favicon.ico
+    run: |
+        cp /stacker/favicon.ico /root/favicon.ico
+EOF
+    cat > img/stacker2.yaml <<EOF
+img2:
+    from:
+        type: docker
+        url: docker://centos:latest
+    import:
+        - https://1ww.cisco.com/favicon.ico
+    run: |
+        cp /stacker/favicon.ico /root/favicon.ico
+EOF
+}
+
+@test "importing non-cached file from http url" {
+    stacker build -f img/stacker.yaml
+    umoci unpack --image oci:img dest/img
+    [ "$(sha reference/favicon_orig.ico)" == "$(sha .stacker/imports/img/favicon.ico)" ]
+    [ "$(sha reference/favicon_orig.ico)" == "$(sha dest/img/rootfs/root/favicon.ico)" ]
+}
+
+@test "importing cached file from http url" {
+    mkdir -p .stacker/imports/img
+    cp reference/favicon_orig.ico .stacker/imports/img/favicon.ico
+    stacker build -f img/stacker.yaml
```

Comment:
Seems like you should grep stacker's output here to make sure it actually uses the cache? Otherwise this doesn't really test anything, right?